File: java/src/main/java/id/my/niznet/tugas1/Main.java
package id.my.niznet.tugas1;

import java.util.ArrayList;

public class Main {
    public static void main(String[] args) {
        ArrayList<Mahasiswa> Mahasiswas = new ArrayList<>();
        ArrayList<Dosen> Dosens = new ArrayList<>();

        String[] mahasiswaNims = {"111", "222", "333", "444", "555"};
        String[] mahasiswaNames = {"Bayu", "Beni", "Reza", "Rahmat", "Roni"};
        String[] mahasiswaJurusan = {"Informatika", "Kimia", "Filsafat", "Biologi", "Matematika"};

        String[] dosenNidns = {"111", "222", "333", "444", "555"};
        String[] dosenNames = {"Agung", "Bobon", "Candra", "Deden", "Endra"};
        String[] dosenSpesialises = {"Rekayasa Perangkat Lunak", "Jaringan Komputer", "Sains Data", "Pemrograman Mobile", "Basis Data"};

        for (int i = 0; i < mahasiswaNims.length; i++) {
            Mahasiswa mhs = new Mahasiswa();
            mhs.setNim(mahasiswaNims[i]);
            mhs.setNama(mahasiswaNames[i]);
            mhs.setJurusan(mahasiswaJurusan[i]);
            Mahasiswas.add(mhs);
        }

        for (int i = 0; i < dosenNidns.length; i++) {
            Dosen dsn = new Dosen();
            dsn.setNidn(dosenNidns[i]);
            dsn.setNama(dosenNames[i]);
            dsn.setSpesialis(dosenSpesialises[i]);
            Dosens.add(dsn);
        }

        for (int i = 0; i < Mahasiswas.size(); i++) {
            Mahasiswa mhs = Mahasiswas.get(i);

            System.out.printf("Object mhs%d%n", i + 1);
            System.out.printf("Nim\t\t: %s%n", mhs.getNim());
            System.out.printf("Nama\t: %s%n", mhs.getNama());
            System.out.printf("Jurusan\t: %s%n", mhs.getJurusan());

            if (i == (Mahasiswas.size() - 1)) {
                break;
            }

            System.out.println("----------------------");
        }

        System.out.println();

        for (int i = 0; i < Dosens.size(); i++) {
            Dosen dsn = Dosens.get(i);

            System.out.printf("Object dsn%d%n", i + 1);
            System.out.printf("Nidn\t\t: %s%n", dsn.getNidn());
            System.out.printf("Nama\t\t: %s%n", dsn.getNama());
            System.out.printf("Spesialis\t: %s%n", dsn.getSpesialis());

            if (i == (Dosens.size() - 1)) {
                break;
            }

            System.out.println("----------------------");
        }
    }
}

File: kotlin/src/main/kotlin/id/my/niznet/tugas1/Main.kt
package id.my.niznet.tugas1

object Main {
    @JvmStatic
    fun main(args: Array<String>) {
        val mahasiswas = ArrayList<Mahasiswa>()
        val dosens = ArrayList<Dosen>()

        val mahasiswaNims = arrayOf("111", "222", "333", "444", "555")
        val mahasiswaNames = arrayOf("Bayu", "Beni", "Reza", "Rahmat", "Roni")
        val mahasiswaJurusans = arrayOf("Informatika", "Kimia", "Filsafat", "Biologi", "Matematika")

        val dosenNidns = arrayOf("111", "222", "333", "444", "555")
        val dosenNames = arrayOf("Asep", "Bobon", "Candra", "Deden", "Endra")
        val dosenSpesialises = arrayOf("Rekayasa Perangkat Lunak", "Jaringan Komputer", "Sains Data", "Pemrograman Mobile", "Basis Data")

        for (i in mahasiswaNims.indices) {
            val mhs = Mahasiswa()
            mhs.nim = mahasiswaNims[i]
            mhs.nama = mahasiswaNames[i]
            mhs.jurusan = mahasiswaJurusans[i]
            mahasiswas.add(mhs)
        }

        for (i in dosenNidns.indices) {
            val dsn = Dosen()
            dsn.nidn = dosenNidns[i]
            dsn.nama = dosenNames[i]
            dsn.spesialis = dosenSpesialises[i]
            dosens.add(dsn)
        }

        for (i in mahasiswas.indices) {
            val mhs = mahasiswas[i]

            println(String.format("Object mhs%d", i + 1))
            println(String.format("Nim\t\t: %s", mhs.nim))
            println(String.format("Nama\t: %s", mhs.nama))
            println(String.format("Jurusan\t: %s", mhs.jurusan))

            if (i == mahasiswas.size - 1) {
                break
            }

            println("----------------------")
        }

        println()

        for (i in dosens.indices) {
            val dsn = dosens[i]

            println(String.format("Object dsn%d", i + 1))
            println(String.format("Nidn\t\t: %s", dsn.nidn))
            println(String.format("Nama\t\t: %s", dsn.nama))
            println(String.format("Spesialis\t: %s", dsn.spesialis))

            if (i == dosens.size - 1) {
                break
            }

            println("----------------------")
        }
    }
}